apps/endatix-hub/app/forms/[formId]/update-form-definition-json.action.ts

```diff
@@ -0,0 +1,13 @@
+"use server";
+
+import { updateFormDefinition } from "@/services/api";
+
+export async function updateFormDefinitionJsonAction(formId: string, formJson: Object | null) {
```

Comment:
Just a mark: this can be better done with `FormUpdateCommand` object. Same for the return value - we can fix this as part of the #124